docusaurus.config.js

```diff
@@ -27,6 +27,13 @@ const docusaurusConfig = {
         },
     ],
     themeConfig: {
+        announcementBar: {
+            id: 'site_announcement', // Any value that will identify this message.
+            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n' + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!'</a>`,
```

Comment:
looks like the single quotes are not needed. 
```suggestion
            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!</a>`,
```